### PR TITLE
Quick change to allow Bulky items and storage containers to be plated.

### PR DESCRIPTION
**Pull request by Momoberry**
I did some testing for bugs, didn't notice any immediately off hand. If anybody remembers any old bugs that occurred, let me know and I'll try and fix em. 

## About the PR 
Changes the Max Weight class to bulky, and allows storage containers to be plated.

## Why's this needed? 
I don't see many exploitable things happening from this off the top of my head, and i think it'd be fun for gimmicks and the like to have goofy backpacks and toolboxes made of weird materials. That and the occasional katana/csaber that blow up or teleports away seems funny. Plates are still blacklisted since they still have the duplication issue with the shards. I've tested the most obvious situations and nothing out of the ordinary has happened. Containers don't plate the items inside, and throwing the plated storage object destroys the objects inside.

## Changelog

```changelog
(u)Moberry
(+)Changed the Electroplater to allow bulky items and storage items.
```

### CODE REVIEW

**Comment by Titanslayer71** [comment]
I will play my singularity buster in frozen farts. Let's see people blast the singularity now!!!

**Review by frawhst — APPROVED** [review]
Might lead to issues but we'll see

File: code/obj/machinery/arc_electroplater.dm [hunk 1]
```diff
@@ -81,7 +81,7 @@
 			boutput(user, "<span class='alert'>That wouldn't possibly fit!</span>")
 			return
 
-		if (W.w_class > src.max_wclass || istype(W, /obj/item/storage/secure) || istype(W, /obj/item/plate)) //can't do plates because of material duping with breaking them over your head
+		if (W.w_class > src.max_wclass || istype(W, /obj/item/storage/secure)) //can't do plates because of material duping with breaking them over your head
```
Comment (TTerc) [inline code/obj/machinery/arc_electroplater.dm:84]: Remove comment pls

Comment (Momoberry) [reply code/obj/machinery/arc_electroplater.dm]: Aye aye 👍

**Pull request merged**